salt/orchestrate/aws/s3_buckets/operations.sls

```diff
@@ -0,0 +1,11 @@
+open-learning-course-data:
+  boto_s3_bucket.present:
+    - Bucket: open-learning-course-data
+    - Versioning:
+        Status: Enabled
+    - region: us-east-1
+    - Tagging:
+        OU: operations
+        business_unit: operations
```

Comment:
Should we have an `open` or `discussions` business_unit and use here?

Reply:
I was wondering about that too and it's probably worth getting @pdpinch's input on the matter.